browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.0",
+  "dependencies": {
+    "@theia/core": "latest",
+    "@theia/editor": "latest",
+    "@theia/filesystem": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/messages": "latest",
+    "@theia/monaco": "latest",
+    "@theia/navigator": "latest",
+    "@theia/preferences": "latest",
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/typescript": "latest",
+    "@theia/workspace": "latest",
+    "che-theia-ssh-extension": "0.0.0"
```

Comment:
I see only theia/core as a CQ
is browser-app dependencies are only build and test dependencies then ?

Reply:
I've created CQ for the entire Theia 0.3.7
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15845